README.md

```diff
@@ -124,6 +161,7 @@ loginsvc:
         private-key-field-name: "privateKey"
         public-key-field-name: "publicKey"
         access-exp-time: 15min
+        key-rotation-time: 9h
```

Comment:
This should be refresh-exp-time not key-rotation time.

In AWS Secrets mode, we don't rotate the key within the app, the scheduler is used to poll aws secrets manager for changes in the secret.
```suggestion
         access-exp-time: 15min
         refresh-exp-time: 9h
         poll-time: 30min
```

Reply:
Done in https://github.com/AbsaOSS/login-service/pull/78/commits/a99fcaaa68b3883208d41e7ca11259fcc12dccee, hopefully as you meant it (sorry, the suggestion box seems a bit confusing)